config/containers/logging/journald.md

```diff
@@ -46,21 +46,24 @@ To configure the logging driver for a specific container, use the `--log-driver`
 flag on the `docker run` command.
 
 ```bash
-$ docker run --log-driver=journald ...
+docker run --log-driver=journald ...
```

Comment:
Can you revert this change (here and others below?) the convention used currently in the docs is to use `$` as prompt in the shell examples 😅

Reply:
Sure, my linter raises it as an issue [no-shell-dollars](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-no-shell-dollars#example) ... and I found it useful when copying the whole paragraph not to elaborate with removing `$` form each line :wink:. Maybe it is worth to consider this in future.